account_invoice_triple_discount/views/account_invoice_view.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="invoice_triple_discount_form_view" model="ir.ui.view">
+        <field name="name">account.invoice.triple.discount.form</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_line_ids']//tree//field[@name='discount']"
+                   position="after">
+                <field name="discount2" groups="base.group_no_one"/>
+                <field name="discount3" groups="base.group_no_one"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="invoice_supplier_triple_discount_form_view" model="ir.ui.view">
+        <field name="name">
+            account.invoice.supplier.triple.discount.form</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_supplier_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_line_ids']//tree//field[@name='discount']"
+                   position="after">
+                <field name="discount2" groups="base.group_no_one"/>
```

Comment:
Why is it only for the debug group?

Reply:
It's the same group as the discount field defined in the account_invoicing module.